impeller/entity/geometry/point_field_geometry.cc

```diff
@@ -28,17 +28,17 @@ GeometryResult PointFieldGeometry::GetPositionBuffer(
     return {};
   }
 
-  Scalar min_size = 1.0f / sqrt(std::abs(determinant));
+  Scalar min_size = 0.5f / sqrt(std::abs(determinant));
```

Comment:
I'm having a brain fart, should this actually be 0.5 / transform.GetMaxBasisLengthXY(); ?

Reply:
I think so. Determinant of 2D matrices is the area of the (transformed) unit square which is pretty much the same thing, but it combines them geometrically so 2x in one direction balances 0.5 in the other. But matrix is a 3D thing so it becomes volume of the unit cube which might not be what we want. Or maybe it's OK since the volume of the cube is the area of the square times the height which is 1.0 - but still it will combine scales between the directions which is not what we want.

So, I think MaxBasisXY is a better value to use.

Really in all of these cases we should be doing separated 2D measurements rather than using the "max" or determinant. We should independently clamp each dimension separately.

Reply:
And if they've imposed a Z scale then determinant would be not at all what we want. Another reason to go with MaxBasisXY

Reply:
Done